lib/ssl/doc/src/ssl.xml

```diff
@@ -1424,10 +1450,14 @@ fun(srp, Username :: string(), UserState :: term()) ->
       <name since="OTP 20.3" name="filter_cipher_suites" arity="2" />
       <fsummary></fsummary>
       <desc><p>Removes cipher suites if any of the filter functions
-      returns false for any part of the cipher suite. This function
-      also calls default filter functions to make sure the cipher
-      suites are supported by crypto. If no filter function is supplied for some
-      part the default behaviour is fun(Algorithm) -> true.</p>
+      returns false for any part of the cipher suite. If no filter function is supplied for some
+      part the default behaviour is fun(Algorithm) -> true.
+
+      Additionaly this function also calls default filter functions to make sure the cipher
+      suites are supported by the cryptolib used by the OTP crypto application.
+      That is calling ssl:filter_cipher_suites(Suites, []) will be equivalent to only
+      applying the filters for cryptolib support.
```

Comment:
I would avoid showing one thing (`fun(Algorithm) -> true`) if the paragraph right after contradicts it since it also mentions.

Maybe just go: "for each part of the algorithm that can be filtered, the default behavior is to make sure the cipher suites are supported by the cryptolib used uby the OTP Crypto application"

Actually now that I write that, is cryptolib support _always_ checked regardless? If so it could be made clearer that the user's filters are only more restrictive. Right now it's unclear whether supplying filters removes that "default behavior" because of the usage of "default" in there.

Reply:
Rewrote it a bit!

Reply:
regard it -> regards it
Otherwise that's much clearer!

Reply:
fixed